whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceAclTestIntegration.java

```diff
@@ -0,0 +1,126 @@
+package net.ripe.db.whois.api.rdap;
+
+import jakarta.ws.rs.ClientErrorException;
+import jakarta.ws.rs.core.MediaType;
+import net.ripe.db.whois.api.rdap.domain.Entity;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.query.acl.IpResourceConfiguration;
+import net.ripe.db.whois.query.support.TestPersonalObjectAccounting;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.net.InetAddress;
+import java.util.ArrayList;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+@Tag("IntegrationTest")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+class RdapServiceAclTestIntegration extends AbstractRdapIntegrationTest {
+
+    private static final String LOCALHOST = "127.0.0.1";
```

Comment:
There is already a WhoisRdapServiceAclTestIntegration . is this doing something different?

Reply:
Ah sorry I see a deleted one

Reply:
Yes I renamed the classes to be prefixed with `Rdap...` for consistency